src/services/events/worker.js

```diff
@@ -78,11 +79,18 @@ const send = async (userId, message, body, id, identityName, setting) => {
 
   if (setting.email) {
     try {
-      await email(body.type, userId, message, id, identityName)
-    } catch(err) {
+      await email(body.type, userId, message, id, identityName, {
```

Comment:
additional kwargs should be be general not only for dispute event worker doesn't need to know what is the args are!

Reply:
Ok let me try to generalize it